relational_operators/CreateIndexOperator.hpp

```diff
@@ -74,7 +86,9 @@ class CreateIndexOperator : public RelationalOperator {
 
  private:
   CatalogRelation *relation_;
+  CatalogDatabase *database_;
   const std::string &index_name_;
+  const std::vector<IndexSubBlockDescription> index_descriptions_;
```

Comment:
I think that using a reference here will not work as the corresponding variable that initializes this, is a stack variable.